dev-util/jenkins-bin/jenkins-bin-2.138.4.ebuild

```diff
@@ -40,7 +40,12 @@ src_install() {
 	newinitd "${FILESDIR}"/${PN}.init2 jenkins
 	newconfd "${FILESDIR}"/${PN}.confd jenkins
 
-	systemd_newunit "${FILESDIR}"/${PN}.service jenkins.service
+	if use systemd ; then
```

Comment:
It's my understanding that systemd unit files should be installed unconditionally even if the system does not have systemd, so these if statements can be removed.

Reply:
@graaff hmm... I activated a IUSE="systemd" whether the system has systemd or not. Generally, systemd-based profiles have 'systemd' globally enabled, and eudev have that disabled
Wouldn't it wiser to condition it, based on USE flag?

Reply:
I think having those files to be installed conditionally is fine since they are not provided by package itself but custom helper files from `${FILESDIR}`

Reply:
@graaff What do you think? I'd say we should keep it as conditional, judging by the fact that it's about our own helpers from `${FILESDIR}` 👍

Reply:
Policy is to install unit files unconditionally.

https://wiki.gentoo.org/wiki/Project:Systemd/Ebuild_policy#General_guidelines

Reply:
@floppym Ah. Alright, will fix this one out and have a discussion on the policy later :)
Thx!

Reply:
@graaff , @floppym Done 👍